cli/src/account.rs

```diff
@@ -17,9 +17,12 @@ pub fn transactions<'a>(client: Client, matches: &'a ArgMatches) -> Result<()> {
     let pager = Pager::from_arg(&matches);
 
     let id = matches.value_of("ID").expect("ID is required");
-    let endpoint = account::Transactions::new(id)
-        .order(ordering::from_arg(matches))
-        .limit(pager.horizon_page_limit() as u32);
+    let endpoint = {
+        let mut endpoint = account::Transactions::new(id)
```

Comment:
Does this actually need to be mutable?

Reply:
I actually don't think so.  Same with transactions.  I'll touch this up